packages/opentelemetry-core/test/platform/BasePlugin.test.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { NoopTracerProvider } from '@opentelemetry/api';
+import { NoopTracerProvider } from '@opentelemetry/api/build/src/trace/NoopTracerProvider';
```

Comment:
does it mean that NoopTracerProvider is not available or it is in different namespace ?

Reply:
The NoopTracer provider is not explicitly exported by the package index so the only way to get it is by its full path. This means it is _not_ part of the API that we claim to be "stable" and could move, change name, be removed, or similar. It also means it will not show up in autocomplete and such when people try to explore the package so it will prevent some confusion.